import-export-cli/docs/apimcli_set.md

```diff
@@ -19,14 +20,17 @@ apimcli set [flags]
 apimcli set --http-request-timeout 3600 --export-directory /home/user/exported-apis
 apimcli set --http-request-timeout 5000 --export-directory C:\Documents\exported
 apimcli set --http-request-timeout 5000
+apimcli set --mode kubernetes
+apimcli set --mode none
```

Comment:
Shall we rename this to `mode normal` 
Mode `none` won't be intuitive. Changing the argument in cobra will do the job

Reply:
@kasvith  thanks for the suggestions. I have changed that to "default" as it makes more sense than "none"

Reply:
+1

Reply:
+1 That looks fine